configs/samples/pjsip.conf.sample

```diff
@@ -658,8 +658,9 @@
                         ; identified.
                         ; "username": Identify by the From or To username and domain
                         ; "auth_username": Identify by the Authorization username and realm
-                        ; "ip": Identify by the source IP address
+                        ; "ip": Identify by the source (remote) IP address
                         ; "header": Identify by a configured SIP header value.
+                        ; "transport": Identfy by the destination (local) IP address
```

Comment:
This doesn't quite do what you document throughout this. It identifies traffic based on the bound address of the transport the traffic is received on. Within signaling that may or may not be the destination address, if such things like NAT are involved.

Reply:
Fixed the description.